### ISSUE TITLE
graffle files for data structures?

### ISSUE THREAD

**Issue by batpigandme** [dcl-docs/prog#15]
@skaltman, just found this the other day and am _loving_ it!
I'm wondering if (like @hadley does) you're using OmniGraffle for the data structure illustrations and, if so, if you'd be willing to share the source files.

For example, this is how Hadley structured things for [Advanced R](https://github.com/hadley/adv-r/tree/master/diagrams). 

Either way, thanks so much for this amazing resource! You've given me many a lightbulb moment!

**Comment by skaltman** [comment]
@batpigandme, yes, we're using OmniGraffle! I'll need to do a bit of file organization, but then I'm happy to put the source files here. 

I'm glad you've found it useful!
